backend/lcfs/web/api/user/services.py

```diff
@@ -332,3 +332,17 @@ async def _has_access_to_user_activities(
 
     async def track_user_login(self, user: UserProfile):
         await self.repo.create_login_history(user)
+
+    @service_handler
+    async def update_notifications_email(self, user_id: int, email: str):
+        try:
+            # Update the notifications_email field of the user
+            return await self.repo.update_notifications_email(user_id, email)
```

Comment:
Can't we pick the user id from the self.request.user object, as we're not allowing users to update for other users, that way we can avoid re-checking the user existence?